### DISCUSSION TITLE
Easier Switching Between Multiple Transmitters

### DISCUSSION THREAD

**Discussion by TomAlperin** [ExpressLRS/ExpressLRS#1333] [Ideas]
### Use Case
User has multiple craft and a few transmitters and wants to be able to switch between transmitters while allowing multiple transmitters to be simultaneously in use.
### Current Behavior
To switch between transmitters that can be simultaneously used, a user must flash the receiver with a new bind phrase. Previous versions allowed the model match number to be changed to allow this but the current version still sets up a connection and just prevents the control signal.
### How This Could Be Implemented
If it is not intended that the model match number be used this way, the bind phrase could instead be stored in a way where it could be modified without flashing firmware. The bind phrase could instead be updated using the wifi page. This would reduce the possibility that a firmware flash would fail disabling a craft out in the field.

**Comment by qubicllj** [comment]
Just don't use bind phrase when flashing receiver, it will be into binding mode after power receiver 3 times quickly.

**Comment by TomAlperin** [comment]
Really like using a bind phrase but will give this a try the next time I flash.

**Comment by Pelado-Mat** [comment]
I wish that the bindphrase could be changed in the WIFI webpage of the RX.

Idea: (don't know if its feasible)
The bindphrase could be changed in the luascript. 
If a RX in bound, it could change the bindphrase in both the TX and RX.

**Reply by brandonrc** [reply]
Think the lua script might be hard because of the scroll wheel on some, but the website idea seems nice. It would be cool to change the bind phrase so you can control someone's quad if they let you borrow it. 

I know I have had to tune a few quads and luckily I have 20 TX's I can just flash to whatever.. but if I only had 1 tx it would be a pain to reflash that one or have to reflash the drone then tell my friend he has to reflash that after I am done.

**Comment by fpvdotkiwi** [comment]
I'd like to expand this to the notion of a bind list.  Up to say.. 16? or so bind phrases, in order of preferred connection. This would surely be enough for anyone.. Then if you have a preference for tx with quad, they're at the top of each others lists. Quads never flown together could have the same bindphrase, etc, lots of flexibility. I know the model match bit exists, but your setup and order of powerup would determine binding automatically with this.

**Comment by Pelado-Mat** [comment]
It seems that there is a PR from @pkendall64  that could partially address this use case, see #1459

**Reply by pkendall64** [reply]
My use case is slightly different, but could be used in this situation if multiple transmitters each have a different bind-phrase. You can easily transfer the model to a different TX by "loan"ing it. You don't actually need to "return" it either. You could just "loan" it out to another transmitter as well! It's just the first "owner" transmitter is special because if you power cycle the RX 3x quickly it will transfer back to the "owner".

**Reply by Pelado-Mat** [reply]
Perhaps a "transfer" option could be doable? The same idea but change the main UID of the RX.

**Reply by pkendall64** [reply]
The main UID is flashed and cannot be changed. What I've done here is to store a secondary UID in the eeprom config section and use that if the model is loaned. If you don't "return" the model either explicitly or power cycling 3x then it's basically transferred anyway.
